test/reducersTest.js

```diff
@@ -0,0 +1,115 @@
+import trumpMM from '../src/Application/Redux/reducers';
+import StateUtils from '../src/Application/Redux/StateUtils';
+import createAction from '../src/Application/Redux/actions';
+import { COLLECT_INCOME } from '../src/Application/Redux/actions';
+import _ from 'underscore';
+
+import chai from 'chai';
+const assert = chai.assert;
+const expect = chai.expect;
+
+describe('reducers', function () {
+
+  describe('general', function () {
+
+    it('should return the initial state given no input state and no action', function () {
+      const initialState = StateUtils.getInitialState();
+      const returnedState = trumpMM(undefined, "");
+      expect(returnedState).to.deep.equal(initialState);
+    });
+
+  });
+
+  describe('Action: CLICK_MONEY', function () {
+
+    it('should leave the original state unmodified', function () {
+      const initialState = StateUtils.getInitialState();
+      const copyOfOriginalState = Object.assign({}, initialState);
+
+      trumpMM(initialState, createAction.clickMoney());
+
+      expect(initialState).to.deep.equal(copyOfOriginalState);
+      expect(initialState).to.have.deep.property('money.cash').that.equals(copyOfOriginalState.money.cash);
+    });
+
+    it('should return a state with 1 more money after a CLICK_MONEY action', function () {
+      const initialState = StateUtils.getInitialState();
+
+      const returnedState = trumpMM(initialState, createAction.clickMoney());
+
+      expect(returnedState).to.have.deep.property('money.cash').that.equals(initialState.money.cash + 1);
+    });
+
+  });
+
+  describe('Action: COLLECT_INCOME', function () {
+
+    it('should leave the original state unmodified', function () {
+      const initialState = StateUtils.getInitialState();
+      const copyOfOriginalState = Object.assign({}, initialState);
+
+      const oneSecondLater = initialState.lastUpdate + 1000;
+      const collectIncomeAction = { type: COLLECT_INCOME, currentTime: oneSecondLater };
+
+      trumpMM(initialState, collectIncomeAction);
+
+      expect(initialState).to.deep.equal(copyOfOriginalState);
+      expect(initialState).to.have.deep.property('money.cash').that.equals(copyOfOriginalState.money.cash);
+    });
+
+    it('should return a state updated lastUpdate field', function () {
```

Comment:
I changed this test, but kept the poorly worded name for legacy purposes. 9/11 never forget.